networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc

```diff
@@ -27,7 +27,7 @@ include::modules/virt-viewing-network-state-of-node-console.adoc[leveloffset=+1]
 // The `NodeNetworkConfigurationPolicy` manifest file
 include::modules/node-network-configuration-policy-file.adoc[leveloffset=+1]
 
-[discrete]
+
 [role="_additional-resources"]
 == Additional resources
```

Comment:
Would using ".Additional resources" going forward be the best solution to avoid cluttered TOCs (level 1 and level 2 headings only)? I like the approach in the using-pipelines-as-code.adoc file that avoids multiple "Additional resources" in the TOC for the Pipelines doc (the file is not perfect, as it's missing the role tag above additional resource entries).

Reply:
@dfitzmau Just to answer your Q here, I don't think that the .Additional resources sections after modules will be supported going forward, so that's definitely not something that we want to introduce more of. Still figuring out more details on this though